core/spree_core.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.files        = Dir['LICENSE', 'README.md', 'app/**/*', 'config/**/*', 'lib/**/*', 'db/**/*', 'vendor/**/*']
   s.require_path = 'lib'
 
-  s.add_dependency 'activemerchant', '~> 1.34'
+  s.add_dependency 'activemerchant', '1.43.3'
```

Comment:
Why did you choose this particular version?

Usually the patch version (last digit) should not matter and we should allow upgrades as long as they don't break anything. In openfoodnetwork we use 1.78.0. With this declaration, there would be a version conflict. What about `~> 1.43`? Would that work?

Every time a work with a Gemfile, I have look up the syntax of `~>` again: https://bundler.io/v1.5/gemfile.html

Reply:
This is the version visible in the Gemfile.lock in Spree repo for branch 2-0-stable. Above versions seem to trigger the error visible in the issue. `~> 1.43` don't work either, same version as earlier (`1.78`) is installed.

Reply:
That's a problem. I searched our commit history and found https://github.com/openfoodfoundation/openfoodnetwork/commit/b5a6bcdf9d540fa0cec7c88911267f3e678e5228 saying:

>  Opening up restriction on activemerchant version by updating Spree ref
>
> Allows use of v > 1.57 which includes support for Stripe Connect

That means we need a newer version of activemerchant. We need to find another solution.